### ISSUE TITLE
Questions about FlownetC

### ISSUE THREAD

**Issue by Blcony** [visinf/irr#13]
Hi ~

Thanks for your wonderful work and codes!

I have re-trained the flownets according to your codes. Though the performance is a little worse than yours results in paper, I thought it is still reasonable. 

Then I tried to re-trained the flownetc, and I found there aren't codes about flownetc, so I wrote the codes by myself according to flownet paper and your pwcnet codes (correlation ops in codes). But I found the loss of flownetc can't decrease after many epoches (10+ epoches). So I am very confused  about this phenomenon.

Therefore, I wonder that: (1) have you tried to train the flownetc according to your own codes? (2) can you give me some suggestions, such as some possible wrong operations, or should I adjust some parts (original learning rate, data augmentation) in codes ?

Your reply will be very helpful for me, and I wish to hear from you soon!

Bset wishes!

Blcony

**Comment by hurjunhwa** [comment]
Hi,

I don't clearly remember, I think I had the same problem before that my implementation of FlowNetC didn't work or at least the loss was not decreasing.
And still I don't have a clear answer.
If you think that the problem comes from the difficult optimization, 
then 1) trying to tune the geometric augmentation parameters a little bit or
2) trying out different initialization methods.
Hope this is helpful, but sorry that I don't have a clear answer :(

**Comment by Blcony** [comment]
Thanks for your reply! 

And I will try your suggestions, and hope I can find the clear answer.

Sorry to bother you again, now I have another question which needs your help,

I replicated the pwcnet according to your codes (in fact I just run the pwcnet.sh), but I obtained the different results compared to yours in table 1 of paper.

I trained the pwcnet  twice, and the setting of them is same, and just a muliti-gpu version of your models. 

The fisrt model's results is shown below, the first row is the performance in paper, the second row is the performance of my replicatation. It seems tha my replication is better than paper's, which is comparable with pwcnet-biocc. But in fact, the setting of seed, dataset and model is just the same, what's the reason causing this phenomenon?  

So I continued to replicate the pwcnet at the second time without any changes. The second model's result on flyingchairsocc is about 1.8473, which is worse than the first one and betther than paper's. 
![image](https://user-images.githubusercontent.com/26156941/68447318-7f843f00-021a-11ea-93e8-6452beea84b4.png)

Based on the experiments, I am confused about the results. So I wonder that (1) Is it the normal phenomenon? Or have I made some mistakes? (2) If it's the normal one, why is the performance of pwcnet so unstable under the same setting (seed, dataset and model) ?

Sorry to bother you again, and I wish to hear from you soon!
 
Thank you very much!

Blcony
